docs/user-guide/images.md

```diff
@@ -100,6 +101,23 @@ Troubleshooting:
   - `plugins.go:56] Registering credential provider: aws-ecr-key`
   - `provider.go:91] Refreshing cache for provider: *aws_credentials.ecrProvider`
 
+### Using Azure Container Registry (ACR)
+When using Azure container registry you can authenticate using either an admin user or a service principal.
+In either case, authentication is done via standard Docker authentication.  These instructions assume the
+python based [az](https://github.com/azure/azure-cli) tooling.
+`
+```console
+# To generate new credentials
+az acr credential renew -n ${CLUSTER_NAME} -g ${CLUSTER_RESOURCE_GROUP}
+
+# To show existing credentials
```

Comment:
It doesn't seem to be rendering this correctly on the netlify preview... https://deploy-preview-1734--kubernetes-io-vnext-staging.netlify.com/docs/user-guide/images/#using-azure-container-registry-acr

I think you need to remove the extra backtick on line 108.